automation/build-bin.ts

```diff
@@ -24,6 +25,43 @@ import { exec as execPkg } from 'pkg';
 import * as rimraf from 'rimraf';
 
 export const ROOT = path.join(__dirname, '..');
+// Note: the following 'tslint disable' line was only required to
+// satisfy ts-node under Appveyor's MSYS2 on Windows -- oddly specific.
+// Maybe something to do with '/' vs '\' in paths in some tslint file.
+// tslint:disable-next-line:no-var-requires
+export const packageJSON = require(path.join(ROOT, 'package.json'));
```

Comment:
You can import json files in typescript now https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-9.html#new---resolvejsonmodule

Reply:
Yes, the feature is even enabled in `automation/tsconfig.json`, I tried it before... Tried again now, today I got:
```
Module '"balena-io/balena-cli/package"' can only be default-imported using the 'esModuleInterop' flagts(1259)
package.json(1, 1): This module is declared with using 'export =', and can only be used with a default import when using the 'esModuleInterop' flag.
```
... and I didn't want to spend more time on this line. :-)